value.go

```diff
@@ -53,6 +53,15 @@ func Null(iso *Isolate) *Value {
 	return iso.null
 }
 
+func NewStringFromByteArray(iso *Isolate, val []byte) (*Value, error) {
+	if iso == nil {
+		return nil, errors.New("v8go: failed to create new Value: Isolate cannot be <nil>")
```

Comment:
This should be treated as a bug, so should instead `panic` to make sure the error is quickly noticed during testing..